### ISSUE TITLE
RPG Skills Feedback

### ISSUE THREAD

**Issue by mtadayz** [NullSystemWorks/mtadayz#158]
Please post feedback, bugs, errors and suggestions here.

**Comment by ghost** [comment]
vendor npc does not work

**Comment by Risusama** [comment]
Not really a giant issue imo, but when upgrading from the previous commit, I could no longer log in to my previously created account, with the following console error:

`[2018-03-12 18:09:55] LOGIN: (Everyone, Admin) DebexeL successfully logged in as 'DebexeL' (IP: 192.168.1.1  Serial: Nope, not going to get it!)
[2018-03-12 18:09:58] WARNING: [DayZ]\DayZ\init\players\login_init.lua:239: Bad argument @ 'fromJSON' [Expected string at argument 1, got boolean]
[2018-03-12 18:09:58] ERROR: [DayZ]\DayZ\init\players\login_init.lua:240: bad argument #1 to 'pairs' (table expected, got nil)`

###EDIT### Removing internal.db fixed the problem. Though, if you do that on a bigger playerbase, it will most likely affects active players as all accounts on the server disappear when you delete this file.

**Comment by mtadayz** [comment]
Did you accept a job prior to logging out?

**Comment by Risusama** [comment]
@mtadayz in case you answered to my comment, no.
I had previously installed the latest files from before this update. I tested the server with my account.
Then I updated to the current latest files. I was trying to test the update with my previous account, the credentials worked, but when I tried to spawn, I only got a black screen and the errors above were in my server console.

**Comment by mtadayz** [comment]
Ah, yes, that's probably because if an existing account tries to log in, it won't have the "PlayerCurrentJob" data, so the login fails. Hopefully fixed it with the latest commit.

**Comment by Risusama** [comment]
Perhaps somehow slightly increase zombie spawnrate near the zombie extermination mission locations? It's kinda very time consuming killing 30 zombies when almost no zombies spawn at the location. I think it should be done (if possible) by upscaling the configured values like in the following example:

Extermination job in San Fierro. With the marker in the center, select a radius from between 100-175meters for which zombie spawnrates will be raised by like, 2 to 3 times the normal. Of course, if doable, the size of the area could also affect the multiplier for the zombie spawnrates.

**Comment by mtadayz** [comment]
Good idea. I wanted to do that before pushing the update, but I forgot about it.

EDIT:
I've added the requested feature, though twice as many zombies will spawn regardless of whether or not you arrived at your destination. Figured the in-game reason for that could be because zombies are actively hunting you down.
